packages/react-avatar/src/components/Avatar/Avatar.types.ts

```diff
@@ -134,24 +135,33 @@ export type AvatarNamedColor =
   | 'anchor';
 
 /**
- * Names of the shorthand properties in AvatarProps
+ * Properties for Avatar
  */
-export type AvatarShorthandPropsCompat = 'label' | 'image' | 'badge' | 'icon';
+export interface AvatarProps extends ComponentProps<AvatarSlots>, Partial<AvatarCommons> {
+  /**
+   * Badge to show the avatar's presence status.
+   * Can either be a string indicating the status ("busy", "away", etc.), or a PresenceBadgeProps object.
+   */
+  badge?: PresenceBadgeStatus | ObjectShorthandProps<PresenceBadgeProps>;
+
+  /**
+   * The Avatar's image.
+   * Can either be the URL of the image, or the props of the img tag
+   */
+  image?: string | ObjectShorthandProps<React.ImgHTMLAttributes<HTMLImageElement>>;
```

Comment:
@behowell this is done to exclude other types from `ShorthandProps` are `img` does not have children, correct?